.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Node.js CI
+
+on: push
```

Comment:
Please apply the CI only on master like below:
```
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Reply:
Why? Shouldn't all commits be tested before merging?

Reply:
Well, yes, but it should be limited to master branch only.

Reply:
What is the problem with testing all branches?